Pod/Classes/VOKCoreDataManager.m

```diff
@@ -6,6 +6,10 @@
 #import "VOKCoreDataManager.h"
 #import "VOKCoreDataManagerInternalMacros.h"
 
+#import <ILGClasses.h>
```

Comment:
How are you developing on this if you haven't `pod install`ed here? I see the addition to the dependency list, but I can't see what else is in this pod.